packages/genesys-spark/package.json

```diff
@@ -9,6 +9,9 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c --watch",
+    "eslint": "eslint --fix './**/*.ts'",
```

Comment:
I tried using `eslint --fix .` here as in other packages, but it wasn't picking up all of the files. It makes me a bit concerned that we might be missing some checks in other places, but I haven't fully looked into it yet.